erpnext_documentation/www/docs/user/manual/en/introduction/index.md

```diff
@@ -1,38 +1,35 @@
 
-- Do accounts to track their sales/purchases.
-- Do their taxes.
-- Pay their employees.
-- Manage deliveries within promised timelines.
-- Deliver quality goods and services.
-- Communicate with customers, answer their queries.
+Whether you are a tiny, small, mid, large, huge, or humongous organization, you will benefit from having an ERP.
 
-Large enterprises invest millions of dollars in highly sophisticated systems like SAP. SAP and similar systems are able to handle requirements from these large enterprises to bring their **multi-country**, **multi-company**, **multi-currency**, global businesses into a single platform. This has helped them achieve consolidation of data/records in the fundamental processes such as accounting, taxation, payroll, reporting, and supply chain to name a few.
+## Why do companies use ERP?
 
-In contrast to large enterprises, SMEs have been left out of the **1st wave of digital revolution (1995-2015)**. The main reason for being left out is because they don't have the financial resources to **license, hire programmers and customize** such software to match their needs. Systems like SAP have been a **no-go** for most SMEs. Such systems need a **minimum** CAPEX or OPEX layout of **$100k (Yes, One Hundred Thousand Dollars)** and upwards. Even with cloud offerings (Software as a Service, SAAS) from such vendors, there is no easy Do It Yourself (DIY) people can feel confident about.
+**1. Single Source of Truth**
 
-So even today, small businesses continue to struggle to keep things organized. They often use a mix of applications and tools such as spreadsheets, accounting software, web-based CRMs to manage their day to day record keeping. As a consequence, they have to access different systems to understand customers' master data, sales, cash flows, or profitability. The information exists in different silos. It becomes hard to combine the data in order to get useful insights or alerts. This leads to SMEs being **reactive** instead of being **proactive** in their day to day operations. With such a constrained way of working, there's limited scope for **growth**.
+If the sales team in your company uses one software to manage sales and the accounting team uses a separate accounting software this could result some issues. The accounting team needs to duplicate the customer data and continuously update it to keep it in sync with the data in the sales software. And depending on whom you ask you may get a different answer for the same question like what is the billing address of a specific customer. ERP systems eliminate this as they act as a single central database (single source of truth). This results in efficiency and data accuracy. You can gain insights into your business via reports and analytics.
```

Comment:
```suggestion
If the sales team in your company uses one software to manage sales and the accounting team uses a separate accounting software this could result in some issues. The accounting team needs to duplicate the customer data and continuously update it to keep it in sync with the data in the sales software. And depending on whom you ask you may get a different answer for the same question like what is the billing address of a specific customer. ERP systems eliminate this as they act as a single central database (single source of truth). This results in efficiency and data accuracy. You can gain insights into your business via reports and analytics.
```